src/nix/profile.md

```diff
@@ -9,10 +9,12 @@ them to be rolled back easily.
 
 # Default profile
 
-The default profile used by `nix profile` is `$HOME/.nix-profile`,
-which, if it does not exist, is created as a symlink to
-`/nix/var/nix/profiles/default` if Nix is invoked by the
-`root` user, or `/nix/var/nix/profiles/per-user/`*username* otherwise.
+The default profile used by `nix profile` is `~/.nix-profile`, unless
+a setting `use-xdg-base-directories` is set to `true`, in which case
+`$XDG_STATE_HOME/ nix/profile` (or `~/.local/state/nix/profile` if
+`XDG_DATA_HOME` is not set) is used. If that profile does not exist,
+it is created as a symlink to `/nix/var/nix/profiles/per-user/
+default` if Nix is invoked by the `root` user, or `/nix/var/nix/profiles/per-user/`*username* otherwise.
```

Comment:
Is there a reason to leave this here? Since we said to rework references to files in a follow-up. Or did you just overlook it?